x-pack/plugins/aiops/kibana.jsonc

```diff
@@ -20,18 +20,14 @@
       "unifiedSearch",
       "share"
     ],
-    "optionalPlugins": [
-      "cases",
-      "observabilityAIAssistant",
-      "usageCollection"
-    ],
+    "optionalPlugins": ["cases", "observabilityAIAssistant", "usageCollection"],
```

Comment:
Is prettier being used to format these files? I only see format changes, so might be worth reverting to original/prettier based formatting.

Reply:
Yeah, this was automatic by Prettier - I think because I **had** changes to that file which were then undone 🤷 I can revert these changes

Reply:
Reverted in c24f29b85c2bcfdb9a5ed57c8ea8c19d7bc91d93 :+1: